react/features/stream-effects/blur/JitsiStreamCropPersonEffect.js

```diff
@@ -0,0 +1,193 @@
+import {
```

Comment:
Is perf okay on mac and windows? I know blur had/has some problem on windows.

Reply:
Performance of crop is same as blur, it will be available only on meet.jit.si as beta, won't be available on 8x8.